src/test/java/com/commercetools/testUtil/customTestConfigs/WebProfileConfiguration.java

```diff
@@ -0,0 +1,71 @@
+package com.commercetools.testUtil.customTestConfigs;
+
+import com.commercetools.pspadapter.APIContextFactory;
+import com.paypal.api.payments.InputFields;
+import com.paypal.api.payments.WebProfile;
+import com.paypal.base.rest.PayPalRESTException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+
+import javax.annotation.PostConstruct;
+import java.util.List;
+
+import static com.commercetools.testUtil.TestConstants.WEB_PROFILE_NO_ADDRESS_OVERRIDE;
+import static java.util.Optional.ofNullable;
+
+/**
+ * Configuration to ensure web profile with {@code address_override == 1} exists.
+ * <p>
+ * After all this configuration injects {@link #noAddressOverrideWebProfile} bean.
+ * <p>
+ * <b>Note:</b> for now this payment integrator does not implement/expose web experience profile services/endpoints,
+ * hence we implement this tests using paypal SDK directly. This might be changed in the future, if the web experience
+ * profile services will be implemented byt this payment integrator.
+ *
+ * @see <a href="https://developer.paypal.com/docs/api/payment-experience/">Payment Experience Web Profiles API</a>
+ */
+public class WebProfileConfiguration {
+
+    @Autowired
+    private APIContextFactory paypalPlusApiContextFactory;
+
+    private WebProfile _noAddressOverrideWebProfile;
```

Comment:
Why `_noAddressOverrideWebProfile` has underscore?

Reply:
to avoid confusion with `noAddressOverrideWebProfile()` bean name